rust/ql/test/query-tests/unusedentities/options.yml

```diff
@@ -0,0 +1 @@
+qltest_cargo_check: false
```

Comment:
I think it is a shame to make all the tests in this folder run without `cargo --check`, as the check is really useful for finding bugs in the test examples. Perhaps create a separate test folder just for unexpandable macros?

Reply:
Done.